xtask/codegen/css.ungram

```diff
@@ -487,13 +487,54 @@ CssDeclaration =
 
 AnyCssProperty =
 	CssGenericProperty
+	| CssComposesProperty
 	| CssBogusProperty
 
 CssGenericProperty =
 	name: AnyCssDeclarationName
 	':'
 	value: CssGenericComponentValueList
 
+// This grammar is specific to CSS Modules and defines how composes classes.
+// https://github.com/css-modules/css-modules/blob/master/docs/composition.md
+// .otherClassName {
+//		composes: globalClassName from global;
+//    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
These two lines are misaligned. It appears that a mix of `spaces` and `tabs` was used here

Reply:
Indeed, thank you!
We need a formatter for `.ungram` :D